composer.json

```diff
@@ -39,13 +39,13 @@
     },
     "require": {
         "php": "^7.2 || ^8.0",
-        "babenkoivan/elastic-adapter": "^1.13"
+        "babenkoivan/elastic-adapter": "^1.16.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
         "orchestra/testbench": "^6.12",
         "babenkoivan/elastic-migrations": "^1.4",
-        "babenkoivan/elastic-scout-driver": "^1.3",
+        "babenkoivan/elastic-scout-driver": "^1.3.1",
```

Comment:
I think we need to require it not only for the dev now, cause the new code will not work with `babenkoivan/elastic-scout-driver` below `1.3.1`.